packages/editor-ui/src/components/CodeNodeEditor/CodeNodeEditor.vue

```diff
@@ -262,19 +341,51 @@ export default defineComponent({
 		extensions.push(this.languageCompartment.of(languageSupport), ...otherExtensions);
 
 		const state = EditorState.create({
-			doc: this.modelValue || this.placeholder,
+			doc: this.modelValue ?? this.placeholder,
 			extensions,
 		});
 
 		this.editor = new EditorView({
 			parent: this.$refs.codeNodeEditor as HTMLDivElement,
 			state,
 		});
+
+		// empty on first load, default param value
+		if (!this.modelValue) {
+			this.$emit('update:modelValue', this.placeholder);
```

Comment:
there might be a bug here or related.. in control, when I add a new code node, default code is not inserted.. only shows up after I open it and close it
<img width="1440" alt="Bildschirmfoto 2023-08-15 um 12 15 48" src="https://github.com/n8n-io/n8n/assets/4711238/a5126e78-24e5-438f-b858-6f12d2425aa9">

Reply:
maybe good to add an E2E test for this since you added tests